specificatie/zoek-reisdocumenten.yaml

```diff
@@ -0,0 +1,125 @@
+              $ref: 'common.yaml#/components/headers/X_Rate_Limit_Remaining'
+            X-Rate-Limit-Reset:
+              $ref: 'common.yaml#/components/headers/X_Rate_Limit_Reset'
+          content:
+            'application/json; charset=utf-8':
+              schema:
+                $ref: '#/components/schemas/ReisdocumentenQueryResponse'
+        '400':
+          $ref: 'common.yaml#/components/responses/400'
+        '401':
+          $ref: 'common.yaml#/components/responses/401'
+        '403':
+          $ref: 'common.yaml#/components/responses/403'
+        '406':
+          $ref: 'common.yaml#/components/responses/406'
+        '415':
+          $ref: 'common.yaml#/components/responses/415'
+        '429':
+          $ref: 'common.yaml#/components/responses/429'
+        '500':
+          $ref: 'common.yaml#/components/responses/500'
+        '501':
+          $ref: 'common.yaml#/components/responses/501'
+        '503':
+          $ref: 'common.yaml#/components/responses/503'
+        'default':
+          $ref: 'common.yaml#/components/responses/default'
+      tags:
+        - Reisdocumenten
+components:
+  schemas:
+    ReisdocumentenQuery:
+      type: object
+      required:
+        - type
+        - fields
+      discriminator:
+        propertyName: type
+        mapping:
+          RaadpleegMetReisdocumentnummer: '#/components/schemas/RaadpleegMetReisdocumentnummer'
+          ZoekMetBurgerservicenummer: '#/components/schemas/ZoekMetBurgerservicenummer'
+      properties:
+        type:
+          type: string
+        fields:
+          type: array
+          maxItems: 130
+          minItems: 1
+          items:
+            $ref: 'filter.yaml#/components/schemas/Field'
+    RaadpleegMetReisdocumentnummer:
+      required:
+        - reisdocumentnummer
+      allOf:
+        - $ref: '#/components/schemas/ReisdocumentenQuery'
+        - type: object
+          properties:
+            reisdocumentnummer:
+              $ref: 'reisdocument.yaml#/components/schemas/Reisdocumentnummer'
+    ZoekMetBurgerservicenummer:
+      required:
+        - burgerservicenummer
+      allOf:
+        - $ref: '#/components/schemas/ReisdocumentenQuery'
+        - type: object
+          properties:
+            burgerservicenummer:
+              $ref: 'persoon.yaml#/components/schemas/Burgerservicenummer'
+    ReisdocumentenQueryResponse:
+      type: object
+      required:
+        - type
+      discriminator:
+        propertyName: type
+        mapping:
+          RaadpleegMetReisdocumentnummer: '#/components/schemas/RaadpleegMetReisdocumentnummerResponse'
+          ZoekMetBurgerservicenummer: '#/components/schemas/ZoekMetBurgerservicenummerResponse'
+      properties:
+        type:
+          type: string
+    RaadpleegMetReisdocumentnummerResponse:
+      allOf:
+        - $ref: '#/components/schemas/ReisdocumentenQueryResponse'
+        - $ref: 'reisdocument.yaml#/components/schemas/Reisdocument'
```

Comment:
ook al staan we nu bij raadplegen maar 1 reisdocument(nummer) toe, toch is het denk ik makkelijker en duidelijker voor een afnemer als we het reisdocument in een envelop "reisdocumenten". te stoppen. Dan houden we de response bij elke zoekvraag gelijk

Reply:
prima. Dit is aangepast. reisdocumentnummer parameter in RaadpleegMetReisdocumentnummer is ook aangepast om het evolvable te maken